repository_service_tuf/cli/admin/ceremony.py

```diff
@@ -149,49 +141,66 @@
 frequently signing a new timestamp.json file with a short expiration time.
 This file indicates the latest version of `snapshot.json`.
 
-Recommended: 1 Key and a Threshold equal to 1
-
-Example:
-Corp Example Inc will use one Key and a Threshold of one Key.
-- DevOps Team
+Uses one single online key,
 """
 
 STEP_1 = """
 # STEP 1: Configure the Roles
 
-The TUF roles supports multiple keys and the threshold (quorum trust)
-defines the minimal number of keys required to take actions using
-a specific Role.
+The TUF root role supports multiple keys and the threshold (quorum of trust)
+defines the minimal number of keys required to take actions using the root role.
+
+Reference: [TUF Goals for PKI](https://theupdateframework.github.io/specification/latest/#goals-for-pki)
+
+The TUF roles have an expiration, clients must not trust expired metadata.
 
-Reference: [TUF](https://theupdateframework.github.io/specification/latest/#goals-for-pki)
+Reference: [TUF expires](https://theupdateframework.github.io/specification/latest/#expires)
 
 """  # noqa
 
 STEP_2 = """
-# STEP 2: Load roles keys
+# STEP 2: Load the Online Key
 
+## The Online Key
+The online key is the same one provided to the Repository Service for TUF
+Workers (RSTUF Worker). This key is responsible for signing the Snapshot,
+Timestamp, Targets, and delegated targets (hash bin) roles.
```

Comment:
Why do you suggest removing the information that clarifies that the online key is the same one used in the worker deployment? Do you think it is not important information for the user? To avoid misunderstanding that they should have another key.
We will also have this described in the Guide, but highlighting it here makes it clear.

Reply:
Ookay, I see what you meant here.
Then the only recommendation I can give is renaming the roles with lowercase as we do in other documentation or:
``` suggestion
The online key is the same one provided to the Repository Service for TUF
Workers (RSTUF Worker). This key is responsible for signing the snapshot,
timestamp, targets, and delegated targets (hash bin) roles.
```